stdlib/config/config.lua

```diff
@@ -0,0 +1,152 @@
+--- Config module
+-- @module Config
+
+require 'stdlib/core'
+require 'stdlib/string'
+require 'stdlib/table'
+
+-----------------------------------------------------------------------
+--Setup repeated code for use in sub functions here
+-----------------------------------------------------------------------
+local reservedCharacters = '`~!@#$%^&*+=|;:/\\\'",?()[]{}<>'
+local testReservedCharacters = function(path)
+    local reservedCharacters = reservedCharacters
+    for c in reservedCharacters:gmatch('.') do
+        if path:find(c, 1, true) then
+            return c
+        end
+    end
+    return nil
+end
+
+if not Config then Config = {} end
```

Comment:
I have no idea why I did that. Especially with the line that follows. Oh the joys of of looking at your code a week later and wondering why you decided something was ever a good idea.